packages/components-native/tsconfig.types.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist/types",
```

Comment:
I think this file can be moved into the one `tsconfig` file using this tsconfig option
https://www.typescriptlang.org/tsconfig#declarationDir

Reply:
I also removed `tsconfig.esm.json`.